src/GlimpseCore.Client/src/shell/themes/views/ThemeButton.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { IconShapeType } from '@common/components/AgentTypeIcon';
+import { Icon } from '@common/components/Icon';
+//import { default as shellStatusBarStyles } from '@shell/views/ShellStatusBarView.scss';
+//import '@shell/views/ShellStatusBarView.scss';
+import shellStatusBarStyles from '@shell/views/ShellStatusBarView.scss';
+
+import { ThemeType } from '../IThemesState';
+
+interface IThemeButtonProps {
+    iconShape: IconShapeType
+    label: string,
+    selectedThemeName: ThemeType,
+    theme: string,
+    onSelectTheme: (theme: string) => void;
+}
+
+export class ThemeButton extends React.Component<IThemeButtonProps, any> {
```

Comment:
If we replace the inline renderButton with a component, we should remove the original use.